experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_k8s_job.yml

```diff
@@ -35,9 +35,9 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
 
-          # Only GCP and AWS is supported
+          # GKE and AWS supported
           - name: CLOUD_PLATFORM
-            value: 'GCP'
+            value: 'GKE'
```

Comment:
Make this empty, Let user fill this

Reply:
I think it is fine to have GKE as default value

Reply:
Default means, if you don't pass any thing. It will used GKE as default. So, it might become ambiguous to the user

Reply:
But I am passing a value `GKE` it means if you will pass GKE then only it will use GKE.

Reply:
But If I will leave it blank then the user may confuse that maybe by default it is taking GKE that's why no value has been provided. I am thinking from that angle. cc: @ksatchit

Reply:
Okay